packages/charts-line/README.md

```diff
@@ -0,0 +1,48 @@
+# IBM Charts - Line
+## @ibm-design/charts-line
+An implementation of the IBM Design Language's [line chart](https://www.ibm.com/design/language/experience/data-visualization/chart-models/linechart/) using [C3.js](http://c3js.org/).
+
+![Alt text](/charts-line.gif?raw=true "Optional Title")
```

Comment:
The link to the gif seems broken when I look at the current branch state: https://github.com/IBM-Design/charts/tree/line-docs/packages/charts-line . Not sure if the `/` at the beginning of the link is needed.